airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -257,6 +257,7 @@ private CancellationScope generateSyncWorkflowRunnable(final ConnectionUpdaterIn
         // silently ignore it.
         if (childWorkflowFailure.getCause() instanceof CanceledFailure) {
           // do nothing, cancellation handled by cancellationScope
+          log.info("");
```

Comment:
Is there is missing log message here? If not, then this should be removed.

Reply:
Sorry I forget to remove it, It was to avoid having a PMD failure.

Reply:
I changed it to a suppress warning, I'll do it in another commit as well.